introduction.md

```diff
@@ -1,3 +1,9 @@
 Watashi wa Taubu no Juryan desu. Douzo yoroshiku onegai itashimasu.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?